### DISCUSSION TITLE
Don't mention "sRGB" in change logs

### DISCUSSION THREAD

**Discussion by matoi974** [obsproject/obs-studio#4533] [General]
The change logs for v27.0 RC1 and RC2 have a mention of "sRGB", here:
![image](https://user-images.githubusercontent.com/63289008/114768325-22cb7300-9d82-11eb-875b-a10ee4e0d141.png)
This will cause a lot of confusion and unnecessary strain on obs-studio's tech support channels when the first release build comes through, just like "new NVENC" once did. Maybe not quite as much, but this is fully preventable and I really hope I won't have to correct the incorrect explanations of countless content creators who will inevitably get it wrong. I would like it to be rephrased to something like this:
 - Added support for applying color operations in linear space [jpark37]

It probably won't make much sense to the average non-tech-savvy person, but the current description will probably mislead them into some wrong conclusion.

**Comment by Fenrirthviti** [comment]
Can you clarify exactly what is going to lead to confusion? There isn't really a tangible concern here other than "users might not know what this is" which is not really something we should be concerned about.

**Reply by matoi974** [reply]
> 
> 
> Can you clarify exactly what is going to lead to confusion? There isn't really a tangible concern here other than "users might not know what this is" which is not really something we should be concerned about.

There's concern among tech support volunteers and some others that this may lead to series of third-party guides with misleading information, which will be annoying to deal with.
Examples: 
https://discord.com/channels/348973006581923840/827748792584503356/831669641159639051
https://discord.com/channels/348973006581923840/827748792584503356/831678375408042014
https://discord.com/channels/348973006581923840/827748792584503356/831672364092882995
